### PR TITLE
Added quick start guide for produce and consume in rhoas cli

### PR DESCRIPTION
**Pull request by jackdelahunt**

### CODE REVIEW

File: docs/rhoas/rhoas-produce-consume/quickstart.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: Produce and consume in Rhoas Cli
+  annotations:
+    draft: false
```
Comment (bhardesty) [inline docs/rhoas/rhoas-produce-consume/quickstart.yml:6]: I would suggest changing this to `draft: true`. That way we can go ahead and merge this PR, and handle the content refinement in a follow-up PR.

**Review by bhardesty — COMMENTED** [review]
Thanks, @jackdelahunt - this looks really helpful, and it's a great start! We'll work to refine the content, but that typically works best as a follow-up PR done by one of the writers.

**Comment by wtrocki** [comment]
Technically looks on. Couple typos that can be fixed after merge. Draft true would allow us to merge it jnow

**Comment by jackdelahunt** [comment]
Updated to be a draft 👍

**Review by bhardesty — APPROVED** [review]

**Pull request merged**